grouping_server/database/serializers.py

```diff
@@ -20,24 +23,25 @@ class Meta:
 class WorkspaceSerializer(serializers.ModelSerializer):
     tags = WorkspaceTagSerializer(
         many=True, required=False, allow_empty=True)
-    photo = ImageSerializer(required=False)
+    photo_data = serializers.ImageField(write_only=True)
+    photo = ImageSerializer(read_only=True)
 
     class Meta:
         model = Workspace
         fields = ['id', 'theme_color', 'workspace_name',
-                  'description', 'is_personal', 'photo', 'members', 'tags', 'activities']
+                  'description', 'is_personal', 'photo_data', 'photo', 'members', 'tags', 'activities']
```

Comment:
I think `is_personal` field is useless in new workspace structure since any workspace is free to join others member.

Reply:
I forgot to remove it, sor.